pkgs/development/python-modules/txaio/default.nix

```diff
@@ -18,7 +18,7 @@ buildPythonPackage rec {
   '';
 
   # Needs some fixing for 3.7
-  doCheck = !isPy37;
+  doCheck = !isPy3k;
```

Comment:
Why did you change this?

Reply:
It didn't pass tests on python 3.8, and `isPy38` doesn't exist.